core/errors.py

```diff
@@ -7,7 +7,10 @@ class ValidationError(RuntimeError):
     def __init__(self, validation_failures: list[ValidationFailure]):
         self.failure_messages = [str(error) for error in validation_failures]
         # TODO: uncomment once all failures have UC messages
-        # self.failure_messages = {"TabErrors": serialise_user_centered_failures(validation_failures)}
+        # if any(isinstance(failure, PreTransFormationFailureClass) for failure in validation_failures):
+        #     self.failure_messages = {"PreTransformationError": serialise_user_centered_failures(validation_failures)}
+        # else:
+        #   self.failure_messages = {"TabErrors": serialise_user_centered_failures(validation_failures)}
```

Comment:
Not a fan of repeating `if any(isinstance(failure, PreTransFormationFailureClass) for failure in validation_failures):` 

I think we just bring the "PreTransformation" and "TabErrors" inside the function.

Reply:
I've brought them inside serialise_user_centered_failures() now